processor_function/shared/service_bus.py

```diff
@@ -0,0 +1,28 @@
+import json
+import os
+from azure.servicebus import ServiceBusClient, ServiceBusMessage
+
+
+class ServiceBus:
+    """
+    Implements methods to operate the service bus in the core infrastructure of Azure TRE.
+    """
+
+    def send_status_update_message(self, id: str, status: str, message: str):
+        """
+        Sends a resource request message for the resource to the service bus
+        :param resource: resource to deploy
+        """
+        resource_request_message = json.dumps({
+            "id": id,
+            "status": status,
+            "message": message
+        })
+
+        service_bus_client = ServiceBusClient.from_connection_string(conn_str=os.environ["servicebusconnection"])
```

Comment:
the processor function does not use MSI to talk to service bus like the API does??

Reply:
No, it's using the same full connection string as with trigger.